### PR TITLE
Mja lb issue seven

### PR DESCRIPTION
**Pull request by arche4**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
Captura

### CODE REVIEW

File: src/components/Product/ListProduct.js [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
+              <tbody>
+                {data.map((value, key) => (
+                  <tr key={key}>
+                    <td>{value.name}</td>
+                    <td>{value.date} </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+            {<Product />}
```
Comment (alejandronanez) [inline src/components/Product/ListProduct.js]: Nicely done @lizhtml @arche4!

You can remove the `{}` from here and just leave the `<Product />` component.

**Review by alejandronanez — APPROVED** [review]
Nicely done @arche4 and @Lizhtml 👯

I tested this in production and works like a charm. 👍 

Also, thank you for deleting those extra files :)

**Review by AndrewEspejo — COMMENTED** [review]
Excellent work @Lizhtml  and @arche4 .  I was able to test it in production and it worked without any problem. 
Congratulations :100:

File: src/components/Home/home.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { Fragment, useState } from 'react';
-import { Link, Redirect } from 'react-router-dom';
```
Comment (AndrewEspejo) [inline src/components/Home/home.js:2]: I like that if you don't use a dependency you delete it to lose time in importing it.

**Review by AndrewEspejo — APPROVED** [review]

File: src/components/Product/ListProduct.js (hunk 1 above)
Comment (Lizhtml) [reply src/components/Product/ListProduct.js]: Now it is ready, thanks.

File: src/components/Product/ListProduct.js [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
+  const [marketListCreated, setMarketListCreated] = React.useState(false);
+
   return (
     <FirestoreCollection
       path={localStorage.getItem('token')}
-      render={({ isloading, data }) => {
-        return isloading ? (
-          <h1>Loading</h1>
+      render={({ data }) => {
+        return !marketListCreated && data.length === 0 ? (
```
Comment (DNeosphere) [inline src/components/Product/ListProduct.js:12]: FAncy 💯

**Pull request merged**